src/components/Calculator.js

```diff
@@ -1,105 +1,204 @@
-/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
```

Comment:
- [ ] Please import operate.js here just like calculate.js for the app to work fine.